caffeine@patapon.info/preferences/generalPage.js

```diff
@@ -27,11 +27,13 @@ const Gettext = imports.gettext.domain(Me.metadata['gettext-domain']);
 const _ = Gettext.gettext;
 const genParam = (type, name, ...dflt) => GObject.ParamSpec[type](name, name, name, GObject.ParamFlags.READWRITE, ...dflt);
 
+/*
 const ComboBoxChoices = {
     NEVER: _('Never'),
     ALWAYS: _('Always'),
     FOR_APPS: _('For apps on list')
 };
+*/
```

Comment:
```suggestion
```

Since this isn't used anywhere else, we probably shouldn't leave commented out code in the codebase

Otherwise, looks good to me